cmd/dp-dataset-exporter/main.go

```diff
@@ -15,12 +15,12 @@ import (
 	filterCli "github.com/ONSdigital/dp-api-clients-go/v2/filter"
 	"github.com/ONSdigital/dp-api-clients-go/v2/health"
 	"github.com/ONSdigital/dp-healthcheck/healthcheck"
-	kafka "github.com/ONSdigital/dp-kafka/v2"
+	kafka "github.com/ONSdigital/dp-kafka/v4"
+	dpotelgo "github.com/ONSdigital/dp-otel-go"
```

Comment:
```suggestion
	dpotel "github.com/ONSdigital/dp-otel-go"
```
including `go` in a go file seems redundant